src/ui/views/mainwindow.cpp

```diff
@@ -352,3 +379,32 @@ void MainWindow::onListRecentAccountsSelectionChanged()
         gtk_list_box_unselect_all(GTK_LIST_BOX(m_listRecentAccounts));
     }
 }
+
+void MainWindow::onListRecentAccountsOnStartSelectionChanged()
+{
+    GtkListBoxRow* selectedRow{ gtk_list_box_get_selected_row(GTK_LIST_BOX(m_listRecentAccountsOnStart)) };
+    if(selectedRow)
+    {
+        std::string path{ adw_action_row_get_subtitle(ADW_ACTION_ROW(selectedRow)) };
+        m_controller.addAccount(path);
+        gtk_list_box_unselect_all(GTK_LIST_BOX(m_listRecentAccountsOnStart));
+    }
+}
+
+void MainWindow::updateStatusPage()
```

Comment:
Since `updateStatusPage()` is only called in `start()`, please move this code inside of the `start()` function and not as it's own function.

Also for the sake of a consistent style, please keep brackets on their own lines i.e.:
```cpp
if
{
}
```

Reply:
Done